sdk/policyinsights/Microsoft.Azure.Management.PolicyInsights/src/Generated/SdkInfo_PolicyInsightsClient.cs

```diff
@@ -24,6 +24,7 @@ internal static partial class SdkInfo
                 new Tuple<string, string, string>("PolicyInsights", "PolicyStates", "2019-10-01"),
                 new Tuple<string, string, string>("PolicyInsights", "PolicyTrackedResources", "2018-07-01-preview"),
                 new Tuple<string, string, string>("PolicyInsights", "Remediations", "2019-07-01"),
+                new Tuple<string, string, string>("PolicyInsights", "PolicyMetadata", "2019-10-01"),
```

Comment:
The header of this file says it's autogenerated, but it wasn't part of the autorest output and we updated it manually. Is it still needed?

Reply:
Hi Elad, please try [updating build tools](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools)

Reply:
Thanks!